tests.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+
+SITE_LOCATION = "http://localhost:5000"
+
+
+def url_ok(url):
+    try:
+        r = urllib.request.urlopen(url).getcode()
+
+    except:
+        r = 404
```

Comment:
its a bad idea to mix error output and real result output.

Reply:
The issue was addressed and the test will only return true or false (true if the connection is established and false if it couldn't and the test found in tests.py will pass or fail accordingly)

Reply:
@ifireball same goes here